tutorials/classical_simulation/Simulating with effective noise channels.ipynb

```diff
@@ -15,25 +15,23 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        print("Noise probability = ", noise_rate)
</pre><p>Missed this one</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/616/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
Yeah turns out I did not save my last modifications before pushing, there are lots of "prob" left...